suite-native/transactions/src/components/TransactionsList/TransactionIconSpinner.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {
+    Canvas,
+    Circle,
+    vec,
+    SweepGradient,
+    useTiming,
+    useComputedValue,
+} from '@shopify/react-native-skia';
+
+import { Box } from '@suite-native/atoms';
+import { useNativeStyles } from '@trezor/styles';
+import { CSSColor } from '@trezor/theme';
+
+type TransactionIconSpinnerProps = {
+    radius: number;
+    color: CSSColor;
+    strokeWidth?: number;
+};
+
+export const TransactionIconSpinner = ({
+    radius,
+    color,
+    strokeWidth = 8,
+}: TransactionIconSpinnerProps) => {
+    const { utils } = useNativeStyles();
+
+    const rotation = useTiming({ from: 0, to: 100, loop: true }, { duration: 2500 });
+    const transformRotate = useComputedValue(
+        () => [{ rotate: ((2 * Math.PI) / 100) * rotation.current }],
+        [rotation],
+    );
+    return (
+        <Box style={{ position: 'absolute' }}>
+            <Canvas style={{ height: radius * 2, width: radius * 2 }}>
```

Comment:
No inline style

Reply:
We agreed that it's OK for Skia only components, if it's simple style.

Reply:
Okay, I didn't know about that. I thought we don't want to use them ever as you commented on some PR before